audformat/core/table.py

```diff
@@ -1169,8 +1172,25 @@ def _save_csv(self, path: str):
         # that is newer than the PKL file
         df = self.df
         with open(path, 'w') as fp:
+            # TODO: Include into code coverage
+            #   once we drop support for < Python 3.8
+
+            tmp_cols = {}
+            for column, dtype in zip(df, df.dtypes):  # pragma: no cover
+                # Since pandas 1.4.0 DataFrame.to_csv()
+                # no longer works for categories with dtype Int64
+                # we have to temporarily convert column to plain Int64
```

Comment:
What's the difference between `dtype Int64` and `plain Int64`?

Reply:
We convert a column with dtype "categories with dtype Int64" to a column with "dtype Int64".